src/wirecraft/server/database/models.py

```diff
@@ -18,3 +22,32 @@ class Device(SQLModel, table=True):
     type: str
     x: int
     y: int
+    id_level: int | None = Field(default=None, foreign_key="level.id")  # Lowercase
+
+
+class Level(SQLModel, table=True):
+    id: int | None = Field(default=None, primary_key=True)
+    completed: bool = False
+
+
+class Task(SQLModel, table=True):
+    id: int = Field(default=None, primary_key=True)
```

Comment:
is default=None the right way to do ?

Reply:
See https://sqlmodel.tiangolo.com/tutorial/automatic-id-none-refresh/

Reply:
Ok, then you should use int | None